data/documentation.yml

```diff
@@ -1,74 +1,75 @@
 toc:
```

Comment:
Why is this file changing so much?

Reply:
oh I didn't realize, it was an autoformatter lol